docs/Documentation/Drivers/Image/Params.md

```diff
@@ -1,2 +1,142 @@
+# VIC Model Parameters
+
 !!! Warning
     Docs not written yet.
+
+# Converting from VIC 4 to VIC 5
+
+We are developing a few scripts to aid users in moving from VIC 4 to VIC 5.  These scripts are currently distributed via [`tonic`](https://github.com/UW-Hydro/tonic).
+
+### Installing Tonic Using Anaconda
+
+The easiest way to install Tonic and its dependencies is to use the [Anaconda Python distribution](https://store.continuum.io/cshop/anaconda/).
+
+To install Anaconda, follow these two simple steps (check to make sure the installer version is the most current)
+
+1.  download and run the Anaconda installer:  [http://continuum.io/downloads](http://continuum.io/downloads)
+
+2.  setup a virtual environment for Tonic
+```shell
+conda create -n tonic anaconda
+source activate tonic
+```
+
+*Note:  you'll need to do the `source activate tonic` to activate the Tonic virtual environment in any new shells.*
+
+Now, download the Tonic source code:
+
+```shell
+git clone git@github.com:UW-Hydro/tonic.git
+```
+
+From the Tonic source code repository, Tonic can be installed using Python's `distutils`:
+
+```shell
+python setup.py install
+```
+
+This installs a top level script, `tonic`, into your bin/ directory and the `tonic` package into your Python path.
+
+If you don't want to use the Anaconda installation I've shown above, you can build the package in your local python installation using:
```

Comment:
By this point, it may be too late, the reader might already have executed the earlier set of commands.  Am I correct in interpreting your instructions to mean that there is a hierarchy of cases (which Anaconda you use, and whether you have write permissions)?  If so, I think you should alert the user to the fact that there are 3 or 4 different courses of action depending on the above conditions - tell them if condition x, go to sub-section 2.x, etc and then label each of these different courses of action with a 2.x sub-heading.